README.md

```diff
@@ -53,6 +53,34 @@ describe("BananService") {
 
 Each `it` block should have only one assertion. This is in line with the _Isolated_ principle.
 
+```swift
+describe("Movie") {
+  context("when initializing from dictionary") {
+  
+    let name = "The Dark Knight Rises"
+    let genre = "Action"
+    let year = 2012
+    let movie: Movie!
+    
+    beforeEach {
+      movie = Movie(dict: ["name": name,
+                           "genre": genre,
+                           "year": year])
+    }
+
+    it("maps name property") {
+      expect(movie.name) == name
+    }
+    it("maps genre property") {
+      expect(movie.genre) == genre
+    }
+    it("maps year property") {
+      epxect(movie.year) == year
+    }
```

Comment:
Yeah, that's another thing. I was always omitting the extra words like e.g. `the` to make the sentence shorter. But as you said, it should read like sentence so it makes sense to include them.